debile/master/server.py

```diff
@@ -215,7 +215,12 @@ def main(args, config):
     if not os.path.isfile(config['keyrings']['pgp']):
         logger.info("Can not find pgp keyring `{file}'".format(file=config['keyrings']['pgp']))
 
-    if args.auth_method == 'ssl':
+    auth_method = args.auth_method
+
+    if config.get('auth_method') is not None:
```

Comment:
The 'auth_method" should be in section 'xmlrpc', like for user.yaml (debile/utils/cli.py).